README.md

```diff
@@ -46,7 +46,7 @@ You’d like to contribute? Don’t hesitate to check out our [contributing guid
 
 ## 📖 Documentation
 
-See our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+See our [Documentation of this SDK](https://meilisearch.github.io/meilisearch-python/), our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
```

Comment:
It's a bit confusing to read :/ Maybe the @meilisearch/docs-team can help out?

Also, the link does not seem to work

Reply:
Good point I will ask the doc-team
Yes the link will work after the release of the next version